src/test/java/redis/clients/jedis/tests/ShardedJedisPoolOverrideObjectFactoryTest.java

```diff
@@ -0,0 +1,63 @@
+package redis.clients.jedis.tests;
+
+import org.apache.commons.pool2.PooledObject;
+import org.apache.commons.pool2.impl.GenericObjectPoolConfig;
+import org.junit.Before;
+import org.junit.Test;
+import redis.clients.jedis.*;
+import redis.clients.jedis.util.Hashing;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.junit.Assert.assertEquals;
+
+public class ShardedJedisPoolOverrideObjectFactoryTest {
```

Comment:
I don't see any necessity of creating a separate test class. Add the test in the existing one.

Reply:
👍

Reply:
done - moved changes to current test class